index.php

```diff
@@ -41,8 +41,8 @@
 
 }
 
-$crawlstart     = $robot->get_crawlstart();
-$crawlend       = $robot->get_last_crawlend();
+$crawlstart     = local_linkchecker_robot\robot\crawler::get_crawlstart();
```

Comment:
I haven't tested this but did you actually try $robot->get_crawlstart() or $robot::get_crawlstart() ? either would seem a bit more readable to me